workflow-varscan/varscan.wdl

```diff
@@ -168,32 +179,76 @@ output {
 }
 
 #=============================================================
-# Task for concatenating mplileups
+# Task for concatenating CNV and SNV variants
 #=============================================================
-task concatMpileup {
+task mergeVariantsNative {
 input {
  Array[File] filePaths
- Int jobMemory = 10
- Int timeout   = 20
+ String outputFile = "concatenated_variants"
+ String outputExtension = "csv"
+ Int jobMemory = 6
+ Int timeout   = 10
 }
 
 parameter_meta {
   filePaths: "Array of pileup files to concatenate"
   jobMemory: "memory in GB for this job"
+  outputExtension: "Extension of the output file"
+  outputFile: "Name of the output file"
   timeout: "Timeout in hours, needed to override imposed limits"
 }
 
 command <<<
```

Comment:
I am afraid, setting pipefail is not going to work - some of those steps (in particular, grep chrM) are Ok to fail since not all runs will have calls for mitochondrial chromosome.